server/t/packets/shutdown.json

```diff
@@ -0,0 +1,6 @@
+{
+    "jsonrpc": "2.0",
+    "id": null,
```

Comment:
This needs an id otherwise the response won't have an id.

Reply:
Please implement as required